deploy_github.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Probably you should call this script `build.sh` since it doesn't deploy, but just builds.

Also `#!/bin/sh` is probably preferable since this also has to run on osx (which I think will soon default to zsh if I recall correctly - don't know if they'll keep bash)

Reply:
Sure